### PR TITLE
Renamed the QgsPenStyleComboBox Qt::NoPen entry to "No Line"

### PR DESCRIPTION
**Pull request by jakimowb**
## Description

All QgsPenStyleComboBox entries mention a "line", with one exception: theentry related to the `Qt::NoPen` enum.

For consistency with other entries, this PR renames "No Pen" to "No Line".

| Before | After |
|---------|------|
| grafik | grafik |

### CODE REVIEW

**Comment by nyalldawson** [comment]
I'm +1 --- @nirvn @DelazJ thoughts?

**Comment by nirvn** [comment]
+1

**Comment by DelazJ** [comment]
+1

**Comment by jakimowb** [comment]
I also tried to update some of the *.ts files accordingly. However, the source text is still "No Pen", although it has already been changed to "No Line" in the "Source and Forms" view. Do you know how I can update the source text?

grafik

**Comment by DelazJ** [comment]
@jakimowb IMHO you shouldn't touch ts files here. they are handled by translators on transifex. It is an automatic process of push/pull and I'm unsure to what extent your changes will be kept at the next pull or will not conflict with what is already in place.
Better join translation team(s) on Transifex if you want to fix/do translations (https://qgis.org/en/site/getinvolved/translate.html#becoming-a-translator).

**Comment by jakimowb** [comment]
@DelazJ it took me a while to find out how to sign-up for an open-source account at https://www.transifex.com/, but yes, now I am waiting for approval.

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**
